packages/core/src/GlobalsHelper.ts

```diff
@@ -1,4 +1,6 @@
-import { encodingForModel } from "js-tiktoken";
+// Note: don't use js-tiktoken it's 60x slower than the WASM implementation
```

Comment:
@himself65 tiktoken supports edge and cloudflare, see https://www.npmjs.com/package/tiktoken